core/src/main/java/io/questdb/cairo/wal/WalEventWriter.java

```diff
@@ -265,6 +296,20 @@ int appendSql(int cmdType, CharSequence sqlText, SqlExecutionContext sqlExecutio
         return txn++;
     }
 
+    int invalidate(boolean invalid, @Nullable CharSequence invalidationReason) {
```

Comment:
This must mark WAL segment as version 1 instead of 0 in WalUtils.WAL_FORMAT_OFFSET_32 offset when this happens.

Reply:
fixed in: https://github.com/questdb/questdb/pull/5507/commits/18dd32480f52eef2f3aac2b537db27d64cbf1cee